testing/keepassxc/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
+# Contributor: August Klein <amatcoder@gmail.com>
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Contributor: Max Rees <maxcrees@me.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=keepassxc
+pkgver=2.2.4
+pkgrel=0
+pkgdesc="KeePassXC is a cross-platform community-driven port of the Windows application Keepass Password Safe."
+url="https://keepassxc.org/"
+arch="all"
+license="GPL-2.0 GPL-3.0 BSD-3-Clause CC0-1.0 LGPL-2.1-only LGPL-3.0-only Nokia-Qt-exception-1.1 MIT BSL-1.0"
```

Comment:
Please use `GPL-2.0+` or `GPL-2.0-or-later` (your preference) for both the GPL2 and GPL3 ones, unless they are GPL-2.0-only and GPL-3.0-only, in which case we cannot accept the package at all as the GPL2 and GPL3 are incompatible with each other when non-upgradeable.

Reply:
The LICENSE.txt (https://github.com/keepassxreboot/keepassxc/blob/148dbc5d41b8f7828c6ae730a92f5d97c13454a7/COPYING) says the art assets keepassxc.png is LGPL-2 (implying only), share/icons/database/* LGPL-2.1 (implying only), share/icons/application/* is LGPL-3.0+.  The code assets are src/streams/qtiocompressor* (case insensitive (LGPL-2 [imply only] or GPL-3; src/gui/KMessageWidget (LGPL 2.1 [implying only with SPDX v2]).

Reply:
The main project is GPL-2 or GPL-3 without plus.

Reply:
In this case, we will take the manual and main.cpp under `GPL-3.0-only`, so the `GPL-2.0` can be removed.

We can take qtiocompressor under `GPL-3.0-or-later`.  The `LGPL-3.0` should become `LGPL-3.0+` or `LGPL-3.0-or-later`.

Reply:
Please update $license accordingly, and I will merge this with the differences in the adelie package and commit it for you.